app/reducers/optional.js

```diff
@@ -1,7 +0,0 @@
-//this is a simple pass thru that offers an extension point
```

Comment:
With your implementation I see that `index.js` has simply been reduced to the export of a JS index. Not exposing the `combineReducers` _might_ be a benefit but if it's a managed file then I would say it's looking like a detriment to clarity. I think most of your audience will be aware of the `combineReducers` part of the API (it gets talked about often) and may well use it downstream in their own reducers so obscuring it here may just be hiding an important detail.

What do you think?

> p.s. I'm still not fully grokking how your "optional.js" is working currently so apologies there